apps/meteor/ee/server/patches/index.ts

```diff
@@ -1,7 +1,10 @@
+import { patchOmniLib } from '@rocket.chat/omni-lib-ee';
+
 import './closeBusinessHour';
 import './getInstanceList';
-import './isDepartmentCreationAvailable';
 import './verifyContactChannel';
 import './mergeContacts';
 import './isAgentAvailableToTakeContactInquiry';
 import './airGappedRestrictionsWrapper';
+
+patchOmniLib();
```

Comment:
This file is always called, I'd say we should call the Omni patches on the ee/livechat-enteprise folder

wdyt?

Reply:
yep, it looks more organized that way